pkg/server/server.go

```diff
@@ -513,6 +516,24 @@ func (h *handler) CallBackForFileLoad(dynamicContent []byte) error {
 	} else {
 		logrus.Infof("BackendMode dynamic file got changed, but same with current mode, skip rebuild mapper")
 	}
+
+	// emit load latency and if first time, skip emit and set to false to avoid instance bounce issue
+	if h.skip {
+		h.skip = false
+	} else {
+		latency, err := fileutil.CalculateTimeDeltaFromUnixInSeconds(backendModes.LastUpdatedDateTime, strconv.FormatInt(time.Now().Unix(), 10))
+		if err != nil {
+			return fmt.Errorf("error parsing latency for dynamic backend mode file: %v", err)
+		}
+		metrics.Get().E2ELatency.WithLabelValues("dynamic_backend_mode").Observe(latency)
+		logrus.WithFields(logrus.Fields{
```

Comment:
instead of logging these, would it be possible to add dimensions on the metrics itself?